api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export function authMiddleware(req, res, next) {
+  /* your code here */
+  const auth = req.headers.authorization;
```

Comment:
Does this work? The usual way of getting headers in express is with `req.get('header-name')`.

I'm not seeing anywhere in the Express docs that says this should work. Are you sure you verified it?

Reply:
I searched StackOverflow originally to find a way to retrieve header from the request.
`req.headers.authorization` seems to be part of the Node HTTP module that Express is built upon.
<img width="1259" alt="Screen Shot 2023-07-20 at 4 11 47 PM" src="https://github.com/loganlee-lth/c0523-code-solutions/assets/36302145/3891d109-437e-4d60-82e0-013f21b3e8dd">

Reply:
I see. Other students have reported that it doesn't work, though maybe they did it wrong. I suggest you use `req.get` since it works even with headers that have special characters, and you can specify both upper and lower case.